readme.md

```diff
@@ -761,6 +761,28 @@ disable_pinch_to_zoom: true # defaults to false
 
 When true, the custom implementations of pinch-to-zoom and double-tap-drag-to-zooming will be disabled.
 
+## show_entity_info_on_itemclick
+
+```yaml
+layout:
+  legend:
+    itemclick: none # disable plotly's normal behaviour of hiding the trace
+show_entity_info_on_itemclick: true # defaults to false
+```
+
+When true, the entity info card will be shown when the legend is clicked
+
+## show_entity_info_on_itemdoubleclick
+
+```yaml
+layout:
+  legend:
+    itemdoubleclick: none # disable plotly's normal behaviour of hiding other traces
+show_entity_info_on_itemdoubleclick: true # defaults to false
+```
+
+When true, the entity info card will be shown when the legend is doubleclicked
+
```

Comment:
@bennop I'm not quite happy with the solution I came up with. This will show the entity info when one clicks/doubleclicks on the trace legend. One also needs to disable the normal plotly behaviour, otherwise the trace will also be hidden, and this won't work if you have only 1 entity in the plot (legend will be hidden by default).

I think your request was quite different, and you expected the info card to show up when the plot has only 1 entity and one clicks somewhere out of the plot, right?

In any case, I need some ideas here.

Reply:
I like the idea of clicking on the legend to perform an action.  The card's default action could be "`toggle-trace`", but be configurable to perform the other standard [dashboard actions](https://www.home-assistant.io/dashboards/actions/#tap_action) (`more-info`, `toggle`, `call-service`, `navigate`, `url`, `none`).  This could be configurable for either the single or double clicks.  Plots with only one entity would need to manually enable showing the legend?

Additionally, from reading plotlys' docs it seems the only usable plotly events for this feature request are `plotly_click` (which is only fired when clicking on data points?) and `plotly_legendclick` but would it be feasible for the card to attach its own event handlers to other elements?  For example, `plot-area`, `title`, `annotation-x`, etc.?

Reply:
That sounds useful :).
There is a bit of a collision with the actual plotly configuration that sets the behaviour of clicking in the legend already, but I think that won't be an issue any more in the new beta. It handles defaults better, and default yaml values  can depend on other parts of the yaml without making a mess in the code.